core/src/main/java/com/linecorp/armeria/server/ServiceConfigBuilder.java

```diff
@@ -106,24 +108,33 @@ public ServiceConfigBuilder decorators(
     }
 
     @Override
-    public ServiceConfigBuilder defaultServiceName(String defaultServiceName) {
-        this.defaultServiceName = requireNonNull(defaultServiceName, "defaultServiceName");
+    public ServiceConfigBuilder defaultLogName(String defaultLogName) {
+        this.defaultLogName = requireNonNull(defaultLogName, "defaultLogName");
         return this;
     }
 
     @Override
-    public ServiceConfigBuilder defaultLogName(String defaultLogName) {
-        this.defaultLogName = requireNonNull(defaultLogName, "defaultLogName");
+    public ServiceConfigBuilder defaultServiceName(String defaultServiceName) {
+        requireNonNull(defaultServiceName, "defaultServiceName");
+        this.defaultServiceName = defaultServiceName;
+        return defaultServiceNaming(ServiceNaming.of(defaultServiceName));
+    }
+
+    @Override
+    public ServiceConfigBuilder defaultServiceNaming(ServiceNaming defaultServiceNaming) {
+        this.defaultServiceNaming = requireNonNull(defaultServiceNaming, "defaultServiceNaming");
         return this;
     }
```

Comment:
```suggestion
    public ServiceConfigBuilder defaultServiceName(String defaultServiceName) {
        requireNonNull(defaultServiceName, "defaultServiceName");
        this.defaultServiceName = defaultServiceName;
        this.defaultServiceNaming = ServiceNaming.of(defaultServiceName);
        return this;
    }

    @Override
    public ServiceConfigBuilder defaultServiceNaming(ServiceNaming defaultServiceNaming) {
        this.defaultServiceName = null;
        this.defaultServiceNaming = requireNonNull(defaultServiceNaming, "defaultServiceNaming");
        return this;
    }
```

Reply:
Thanks for a good suggestion. It helps to keep consistency between them.